BedrockServer.cpp

```diff
@@ -268,13 +268,9 @@ void BedrockServer_WorkerThread(void* _data) {
 
 // --------------------------------------------------------------------------
 BedrockServer::BedrockServer(const SData& args)
-    : STCPServer(""), _replicationState(SQLC_SEARCHING), _replicationCommitCount(0), _nodeGracefulShutdown(false),
-      _masterVersion("") {
-    // Initialize
-    _args = args;
-    _requestCount = 0;
-    _suppressCommandPort = false;
-    _suppressCommandPortManualOverride = false;
+    : STCPServer(""), _args(args), _requestCount(0), _writeThread(nullptr), _replicationState(SQLC_SEARCHING),
```

Comment:
@tylerkaraszewski What do you think of getting more rigorous about the use of constructor initializer lists?